.github/workflows/remove-inactive-label.yml

```
@@ -0,0 +1,27 @@
+name: Remove the inactive label
+
+on:
+  discussion_comment:
+    types: [created]
+
+jobs:
+  build:
```

Could you please use a more descriptive name for jobs?

```suggestion
  remove-label:
```

Otherwise, users get failures like this:

https://github.com/community/community/actions/runs/18106131624

> [build](https://github.com/community/community/actions/runs/18106131624/job/51521056989#step:5:19)
Process completed with exit code 1.